app/lib/aws/package.scala

```diff
@@ -0,0 +1,14 @@
+package lib
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{AWSCredentialsProviderChain, EnvironmentVariableCredentialsProvider, InstanceProfileCredentialsProvider}
+
+package object aws {
+  val ProfileName = "membership"
+
+  lazy val CredentialsProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider(ProfileName),
+    new InstanceProfileCredentialsProvider(false),
+    new EnvironmentVariableCredentialsProvider()
```

Comment:
I don't think you'll need the EnvironmentCredentialsProvider, I added that to work with Lambdas where credentials are provided in the runtime environment. Not that it will hurt having it there!